components/econet/econet.h

```diff
@@ -114,6 +126,7 @@ class Econet : public Component, public uart::UARTDevice {
   std::queue<std::string> datapoint_ids_for_read_service_;
 
   uint32_t read_requests_{0};
+  uint32_t loop_now_{0};
   uint32_t last_request_{0};
```

Comment:
This is used now in the latest commit. We could use millis() through out. But I chose to set this at the start of the loop we are in so that its not changing throughout the loops execution and reference this.